suites/tipset/rewards_penalties.go

```diff
@@ -201,9 +201,7 @@ func TipSetTest_MinerRewardsAndPenalties(t *testing.T, factory state.Factories)
 		gasLimit := gasPenalty - 130
 
 		bb.WithBLSMessageAndCode(
-			td.MessageProducer.Transfer(builtin.BurntFundsActorAddr, alice,
-				chain.Nonce(1), // cause the message application to fail resulting in a miner penalty.
```

Comment:
Lost a useful comment here and below

Reply:
*shakes fist at goland's rafactor feature*, nice catch will fix.